pkg/util/nsenter/nsenter.go

```diff
@@ -122,3 +123,25 @@ func (ne *Nsenter) SupportsSystemd() (string, bool) {
 	systemdRunPath, hasSystemd := ne.paths["systemd-run"]
 	return systemdRunPath, hasSystemd
 }
+
+// HostPath returns the path name on the host after evaluating symlinks on the
+// host.
+func (ne *Nsenter) HostPath(pathname string) (string, error) {
```

Comment:
I'd prefer some EvalSymlinksOnHost (or just EvalSymlinks), because that's what this function does.